.github/workflows/conformance-ginkgo.yaml

```
@@ -497,7 +560,7 @@ jobs:
 
   merge-upload-and-status:
     name: Merge Upload and Status
-    if: ${{ always() }}
+    if: ${{ always() && !inputs.is-workflow-call }}
```

Can we just match on the event_name? This eliminates the possibility a caller overrides this in the inputs:
```suggestion
    if: ${{ always() && github.event_name != 'workflow_call' }}
```

Then we can also delete this input from the top of the file.

(Incidentally I'm not sure I exactly understand why `workflow_call` jobs should not report common post-job outputs, shouldn't this report failures, upload sysdump, or do other steps that developers might find useful?)

---

Ah answer to my last part: This step is performed in the calling workflow, so no need to trigger it here. Probably a comment could help to clarify that.

---

:face_with_head_bandage: not doable due to https://github.com/cilium/cilium/pull/41541#discussion_r2378782808 .